skopt/tests/test_space.py

```diff
@@ -160,6 +160,7 @@ def test_space_consistency():
     s3 = Space([Real(0, 1)])
     s4 = Space([(0.0, 1.0)])
     s5 = Space([(0.0, 1.0, "uniform")])
+    s6 = Space([(0, 1.0)])
```

Comment:
haha no, I meant to test the behaviour that both of them are treated as Categorical, (the current behaviour on master breaks)